docker/README.md

```diff
@@ -9,10 +9,28 @@ Apache Bookkeeper is a software project of the Apache Software Foundation, provi
 # How to use this image
 
 Bookkeeper needs [Zookeeper](https://zookeeper.apache.org/) in order to preserve its state and publish its bookies (Bookkeeper servers). The client only need to connect to a Zookeeper server in the ensamble in order to obtain the list of Bookkeeper servers.
+## standalone BookKeeper cluster
+Just like running a BookKeeper cluster in one machine(http://bookkeeper.apache.org/docs/latest/getting-started/run-locally/), you can run a standalone BookKeeper in one docker container, the command is:
+```
+docker run -it --env JAVA_HOME=/usr/lib/jvm/jre-1.8.0 \
+--entrypoint "/bin/bash" apache/bookkeeper:4.5.0 \
```

Comment:
To be align with other command, it may be better to put "apache/bookkeeper:4.5.0" at the end of this command.